lib/src/components/NxGlobalSidebar/NxGlobalSidebar.scss

```diff
@@ -149,3 +154,52 @@ $nx-global-sidebar-separator-color: $nx-grey-900;
     visibility: hidden;
   }
 }
+
+.nx-global-sidebar__other-content {
+  @include container-vertical;
+
+  padding-bottom: $nx-spacing-md;
+
+  + .nx-global-sidebar__footer {
+    border-top: 1px solid $nx-global-sidebar-separator-color;
```

Comment:
Is there a reason this isn't a bottom border on the `.nx-global-sidebar__other_content`?

Reply:
> > Is there a reason this isn't a bottom border on the .nx-global-sidebar__other_content?
> 
> I guess the `other-content` doesn't necessarily expand to where this border would be

The other issue is that the border is only present when there is `other-content` followed by a footer.